spec/features/navigation_spec.rb

```diff
@@ -70,6 +74,10 @@
       end
 
       it "can redirect an admin to 404 error if they try to access any path with /admin" do
+        within 'nav' do
+          click_link "Login"
+        end
+
```

Comment:
Yup, as you said, definitely a good spot to refactor when we clean up this feature branch today!